### DISCUSSION TITLE
Feed Sync Issues

### DISCUSSION THREAD

**Discussion by immauss** [immauss/openvas#264] [Announcements]
All,
         Please know that there is currently an issue with Greenbone's Feed sync system. This issue cause the initial feed-sync on container start to fail, which causes the container to fail. Until they workout the issue, you can start the contianer using: 

``` -e SKIPSYNC=true``` 

Until they manage to workout their issues, I'll shift my cron to rebuild the image on a daily basis. To keep track of what's going on, you can follow the thread over on their forum:
         https://forum.greenbone.net/t/community-feed-connection-reset-by-peer/17136
         
There is a refresh running now to update the feeds on the public image on docker hub, so it should be up-to-date with the hour. 
         
Thanks,
 Scott

**Comment by vg-arunkumar-adhipathi** [comment]
Hi Scott,

Can you please help me with command to start the container with sync command

**Comment by immauss** [comment]
There are lots of options, but you can find the full docs here: 
https://immauss.github.io/openvas/
 OR
For a quick start, without persistent storage and no synchronization:
```
docker run --detach --publish 8080:9392 -e SKIPSYNC=true --name openvas immauss/openvas
```

This should start up and be accessible in a few minutes on port 8080, and ready to scan in under 15 minutes. The default credentials are admin:admin,  so make sure you change it right away. 

-Scott

**Comment by vg-arunkumar-adhipathi** [comment]
Thank you so much, it seems the issue resolved.

Now we can access the openvas.

Regards,
Arunkumar A

**Comment by tpaiii3** [comment]
Hi, This issue still be active, right?

I cannot restart the existing container as picture below.

![image](https://github.com/immauss/openvas/assets/85722230/80b55a70-2cf5-4f32-8042-7d02901d1f0c)

**Comment by immauss** [comment]
@tpaiii3 
This happens from time to time with GB. Usually it's transient and resolves on it's own. 
You can always pass the "SKIPSYNC=true" option to the container, and it will bypass the sync and start up. 

-Scott

**Comment by juanmachadoathens** [comment]
This problem is still active as of 7/15/2024, right?

Trying to acquire lock on /var/lib/openvas/feed-update.lock
Acquired lock on /var/lib/openvas/feed-update.lock
⠏ Downloading Notus files from rsync://feed.community.greenbone.net/community/vulnerability-feed/22.04/vt-data/notus/ to /var/lib/notus
rsync: [Receiver] failed to connect to feed.community.greenbone.net (45.135.106.143): Connection timed out (110)
rsync: [Receiver] failed to connect to feed.community.greenbone.net (2a0e:6b40:20:106:20c:29ff:fe7f:d2ae): Cannot assign requested address (99)
rsync error: error in socket IO (code 10) at clientserver.c(139) [Receiver=3.2.7]

⠇ Downloading NASL files from rsync://feed.community.greenbone.net/community/vulnerability-feed/22.04/vt-data/nasl/ to /var/lib/openvas/plugins
rsync: [Receiver] failed to connect to feed.community.greenbone.net (45.135.106.143): Connection timed out (110)
rsync: [Receiver] failed to connect to feed.community.greenbone.net (2a0e:6b40:20:106:20c:29ff:fe7f:d2ae): Cannot assign requested address (99)
rsync error: error in socket IO (code 10) at clientserver.c(139) [Receiver=3.2.7]

Releasing lock on /var/lib/openvas/feed-update.lock

Trying to acquire lock on /var/lib/gvm/feed-update.lock
Acquired lock on /var/lib/gvm/feed-update.lock
⠇ Downloading SCAP data from rsync://feed.community.greenbone.net/community/vulnerability-feed/22.04/scap-data/ to /var/lib/gvm/scap-data
rsync: [Receiver] failed to connect to feed.community.greenbone.net (45.135.106.143): Connection timed out (110)
rsync: [Receiver] failed to connect to feed.community.greenbone.net (2a0e:6b40:20:106:20c:29ff:fe7f:d2ae): Cannot assign requested address (99)
rsync error: error in socket IO (code 10) at clientserver.c(139) [Receiver=3.2.7]

⠇ Downloading CERT-Bund data from rsync://feed.community.greenbone.net/community/vulnerability-feed/22.04/cert-data/ to /var/lib/gvm/cert-data
rsync: [Receiver] failed to connect to feed.community.greenbone.net (45.135.106.143): Connection timed out (110)
rsync: [Receiver] failed to connect to feed.community.greenbone.net (2a0e:6b40:20:106:20c:29ff:fe7f:d2ae): Cannot assign requested address (99)
rsync error: error in socket IO (code 10) at clientserver.c(139) [Receiver=3.2.7]

⠇ Downloading gvmd data from rsync://feed.community.greenbone.net/community/data-feed/22.04/ to /var/lib/gvm/data-objects/gvmd/22.04
rsync: [Receiver] failed to connect to feed.community.greenbone.net (45.135.106.143): Connection timed out (110)
rsync: [Receiver] failed to connect to feed.community.greenbone.net (2a0e:6b40:20:106:20c:29ff:fe7f:d2ae): Cannot assign requested address (99)
rsync error: error in socket IO (code 10) at clientserver.c(139) [Receiver=3.2.7]

Releasing lock on /var/lib/gvm/feed-update.lock

**Comment by immauss** [comment]
@juanmachadoathens 
Not that I'm aware. Please verify your connectivity and keep trying. It does happen from time to time that there are connectivity issues with GB, but it is not an issue with the contianer. 

-Scott

**Comment by Mxlkommarz** [comment]
Hi I’m new to the openvas and  I want to fully  understand a few things 
I downloaded open as in Kali and with a successful installation but I’m having issues it says ”no scap database found” in the log I have updated it so many times with some successful and others not but it still persists 
 I successfully got the scan config options in the web but when I scan it queues and still no scab data in CVE and other options 
How do I solve the issue ?

**Comment by immauss** [comment]
@Mxlkommarz 
Let's create a new issue as this one is rather old . . . 

https://github.com/immauss/openvas/issues/306
